tessera-context/src/main/java/com/quorum/tessera/context/RuntimeContext.java

```diff
@@ -37,6 +37,10 @@
 
   boolean isOrionMode();
 
+  boolean getOutputServerURIs();
```

Comment:
These 2 properties (arguably only 1 is necessary) might not be needed in the RuntimeContext